runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx

```diff
@@ -7,13 +7,17 @@ import Image from 'next/image';
 
 import { ContentfulRichText } from '#contentful/common';
 
+import EmbeddedVideo from '../EmbeddedVideo';
 import Typography from '../Typography';
 import styles from './RichTextRenderer.module.scss';
 
+
 interface Props {
     content: ContentfulRichText;
 }
 
+const rex = new RegExp('\\$https:\\/\\/www\\.youtube\\.com\\/watch\\?v=[^$]*\\$', 'g');
```

Comment:
![image](https://github.com/runbotics/runbotics/assets/44121335/600ea7c4-016f-4ba6-bad4-338dc528b9e6)

create a more appropriate name for the variable